libretro-common/string/stdstring.c

```diff
@@ -631,87 +630,3 @@ int find_index_substring_string(const char* str1, const char* str2)
 
    return -1;
 }
-
```

Comment:
Oh, these were intentionally broken out from the string functions and placed back in the tasks.  Roger.

Reply:
Moved the 2 cue-related functions to the cue file, since it doesn't really fit in with the rest of the libretro-,common string functions. They're more serial scanning and manipulation.